### PR TITLE
Separation of the deployment from the monitoring solution

### PR DESCRIPTION
**Pull request by gabrielifg**
* **Please check if the PR fulfills these requirements**
  * [ ] Tests for the changes have been added (for bug fixes / features)
  * [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

Feature and docs update

* **What is the current behavior?** (You can also link to an open issue here)

Currently, the monitoring solution is being initialized along with the platform's standard services.

* **What is the new behavior (if this is a feature change)?**

Now, to make the platform even more modular, the monitoring solution will be deployed separately and according to users' needs.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

No

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)

* **Other information**:

### CODE REVIEW

File: deploy-monitoring.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Metrics (Prometheus, Grafana, Loki, Promtail and Node Exporter)
+- hosts: dojot-k8s[0]
+  any_errors_fatal: true
+  gather_facts: true
+  tags:
+    - metrics
+    - 100k
```
Comment (marianoleonardo) [inline deploy-monitoring.yaml:7]: Acho que poderia remover a tag do `100k`, visto que estamos desacoplando o playbook

File: roles/metrics/README.md [hunk 2]
```diff
@@ -180,6 +179,20 @@ kubectl describe service prometheus-server -n dojot-monitoring
 
 * After that, just open an ssh tunnel with the endpoint obtained, for example:
 
+## How to deploy monitoring solution?
+
+Initially, it is necessary to instantiate the volumes that will be used by the services to store and obtain log data, metrics and others. Given that, run the command below to instantiate the volumes:
```
Comment (marianoleonardo) [inline roles/metrics/README.md:184]: Eu trocaria o `Given that` por `So`.

Acho que poderia trocar também `instantiate` por `deploy`

File: roles/metrics/README.md [hunk 3]
```diff
@@ -180,6 +179,20 @@ kubectl describe service prometheus-server -n dojot-monitoring
...
+
+```
+ansible-playbook -K -k -u gbessa -i inventories/example_local/ volume-monitoring.yaml
```
Comment (marianoleonardo) [inline roles/metrics/README.md:187]: Alterar o usuário de `gbessa` para `dojot`

File: roles/metrics/README.md [hunk 4]
```diff
@@ -180,6 +179,20 @@ kubectl describe service prometheus-server -n dojot-monitoring
...
+```
+
+Now that the volumes are available, we can deploy the monitoring services by running the command below:
+
+```
+ansible-playbook -K -k -u gbessa -i inventories/example_local/ deploy-monitoring.yaml
```
Comment (marianoleonardo) [inline roles/metrics/README.md:193]: Alterar o usuário de `gbessa` para `dojot`

File: volume-monitoring.yaml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+      tags:
+        - prometheus
+    - role: volumes/loki
+      tags:
+        - loki
+    - role: volumes/grafana
+      tags:
+        - grafana
+
+- hosts: master_nodes
```
[the remark this answers is not in the document]
Comment (gabrielifg) [reply volume-monitoring.yaml:18]: Dani, a ideia é que a parte de metrica seja complementar, então é necessário ter ao menos o "core" da dojot já em execução.

File: roles/metrics/templates/prometheus/prometheus-values.yml.j2 [hunk 6]
```diff
@@ -88,15 +88,15 @@ server:
     accessModes:
       - ReadWriteOnce
     size: 1Gi
-    mountPath: /data
-    storageClass: "{{ dojot_storage_class_name }}"
+    mountPath: {{ dojot_volume_directory }}/prometheus
+    storageClass: "{{ monitoring_storage_class_name }}"
```
Comment (danielapetito) [inline roles/metrics/templates/prometheus/prometheus-values.yml.j2:92]: não seria dojot_storage_class_name  ?

**Review by danielapetito — APPROVED** [review]

**Review by marianoleonardo — APPROVED** [review]

**Pull request merged**